gqlalchemy/query_builder.py

```diff
@@ -477,6 +478,30 @@ def construct_query(self) -> str:
         return f"{self.custom_cypher}"
 
 
+class ForeachPartialQuery(PartialQuery):
+    def __init__(self, variable: str, expression: str, update_clauses: str):
+        super().__init__(DeclarativeBaseTypes.FOREACH)
+        self._variable = variable
+        self._expression = expression
+        self._update_clauses = update_clauses
+
+    @property
+    def variable(self) -> str:
+        return "" if self._variable is None else self._variable
+
+    @property
+    def expression(self) -> str:
+        return "" if self._expression is None else self._expression
+
+    @property
+    def update_clauses(self) -> str:
+        return "" if self._update_clauses is None else self._update_clauses
```

Comment:
We have discussed this `None` -> "" conversion. We should  keep this in constructor. If it can't be `None`, remove this conversion